// Throw an exception with a custom message
throw:Something went wrong
